src/parser.y

```diff
@@ -210,31 +211,40 @@ assignment:
                 bool found = false;
                 for (auto &assignment : scope_stack.top()->assignments) {
                     if (assignment.name == $1) {
-
-                        auto RootFile = rootmodule->getFullpath();
+                        auto MainFile = (*mainFile).string();
```

Comment:
It nicer to just use the arrow notation; `mainFile->string()`. .. if we even need to use a pointer (see below)